hscontrol/app.go

```diff
@@ -117,32 +117,32 @@ func NewHeadscale(cfg *types.Config) (*Headscale, error) {
 	}
 
 	var dbString string
```

Comment:
nit: If we were to implement the method to get the db string on the `DatabaseConfig`, we can just invoke 
```
dbString, err := cfg.Database.GetDBConnectionString()
if err != nil {
  return nil, errors.Wrap(err, "failed to get connection string")
}
```